CHANGELOG.md

```diff
@@ -1,5 +1,37 @@
+* No new clusters can be created with [Azure AD Integration (legacy)](https://learn.microsoft.com/azure/aks/azure-ad-integration-cli). Existing AKS clusters with Azure Active Directory integration will keep working. All Azure AD Integration (legacy) AKS clusters will be migrated to [AKS-managed Azure AD](https://learn.microsoft.com/azure/aks/managed-azure-ad) automatically starting from December 1st, 2023. We recommend updating your cluster with AKS-managed Azure AD before December 1st, 2023. This way you can manage the API server downtime during non-business hours.
+* Starting January 2024, due to Gatekeeper Upstream removing validation for constraint template contents at create/update time, [the Azure Policy Add-On](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fazure%2Fgovernance%2Fpolicy%2Fconcepts%2Fpolicy-for-kubernetes%23install-azure-policy-add-on-for-aks&data=05%7C01%7Cnehakulkarni%40microsoft.com%7C132aa96f5b9a4108065908dbd718d258%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638340273857129567%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=73Ow8x1Ox8bTw5yOP5P4NIk1R%2FYnaDzhExBBuDbz62g%3D&reserved=0) will now no longer support this. The Azure Policy Add-On will report [‘InvalidConstraint/Template’ compliance reason code](https://learn.microsoft.com/en-us/azure/governance/policy/how-to/determine-non-compliance#aks-resource-provider-mode-compliance-reasons) for detected errors after constraint template admission. This change does not impact [other compliance reason codes](https://learn.microsoft.com/en-us/azure/governance/policy/how-to/determine-non-compliance#aks-resource-provider-mode-compliance-reasons). Customers are encouraged to continue to follow best practices when updating Azure Policy for Kubernetes definitions (i.e. [Gator CLI](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fopen-policy-agent.github.io%2Fgatekeeper%2Fwebsite%2Fdocs%2Fgator%2F&data=05%7C01%7Cnehakulkarni%40microsoft.com%7C132aa96f5b9a4108065908dbd718d258%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638340273857147647%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=6VrGYNGtikDsPw3odmyR9WatG4Pe1tlsvDgHNg85eZY%3D&reserved=0) and [Azure Policy safe deployment features](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fazure%2Fgovernance%2Fpolicy%2Fhow-to%2Fpolicy-safe-deployment-practices&data=05%7C01%7Cnehakulkarni%40microsoft.com%7C132aa96f5b9a4108065908dbd718d258%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638340273857154404%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=AmtvQrzUxz2F6xOJ76%2Bl6zciYrz%2B%2BagUz36G4HKYbdI%3D&reserved=0)).
```

Comment:
Looks like this is an outlook link - might want to change it to: https://learn.microsoft.com/azure/governance/policy/concepts/policy-for-kubernetes#install-azure-policy-add-on-for-aks:~:text=exception%20YAML.-,Install%20Azure%20Policy%20Add%2Don%20for%20AKS,-Before%20you%20install